litellm/integrations/opik.py

```diff
@@ -0,0 +1,199 @@
+from typing import Any, Dict, Callable
+import litellm
+from litellm.types.utils import ModelResponse
+import datetime
+
+"""
+Opik Logger that logs LLM events to an Opik server
+"""
+
+BLUE = "\033[94m"
+RED = "\033[91m"
+RESET = "\033[0m"
+
+def pformat(text: str, level: str="info") -> str:
+    """
+    Format the text with colors.
+
+    Args:
+        text: the text to format
+        info: the mode ("input" or "error")
+
+    Returns a formatted string
+    """
+    return "%s%s%s" % (BLUE if level == "info" else RED, text, RESET)
+
+def get_current_span_id(metadata: Dict[str, Any]) -> str:
+    from opik.opik_context import get_current_span_data
+
+    if metadata.get("current_span_data"):
+        current_span_data = metadata.get("current_span_data")
+    else:
+        current_span_data = get_current_span_data()
+
+    if current_span_data:
+        return current_span_data.id
+    else:
+        return None
+
+
+def get_current_trace_id(metadata: Dict[str, Any]) -> str:
```

Comment:
these are utils functions. Please make a folder called optik/ 

use this structure:

optik/utils.py -> place optik utils here 
optik/optik.py -> place main logging code here